common/djangoapps/course_modes/api/serializers.py

```diff
@@ -28,6 +28,10 @@ class CourseModeSerializer(serializers.Serializer):
     sku = serializers.CharField(required=False)
     bulk_sku = serializers.CharField(required=False)
 
+    class Meta(object):
+        # For disambiguating within the drf-yasg swagger schema
+        ref_name = 'course_modes-CourseMode'
```

Comment:
Curious: Is there a reason why the namespacing is separated by hyphens instead of dots?

Reply:
AFAICT, these names are used only to name cross-references within the .yaml file and don't affect the resulting schema at all.  We can use whatever punctuation we want, but I don't think it matters.  Now that you mention it, a dot would be more natural.  I can change them.

Reply:
Changed to dot.